honestcash-v2/src/app/modules/editor/header/header.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { HeaderComponent } from './header.component';
+
+describe('HeaderComponent', () => {
+  let component: HeaderComponent;
+  let fixture: ComponentFixture<HeaderComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+                                     imports: [],
```

Comment:
Formatting is off.

Reply:
the header itself is still in progress. the test itself has not yet been touched. You comment on stuff that are kind of irrelevant. for example you don't comment why header.component is inside the editor folder but rather in the shared folder. Please as I have OCD, do not comment for formatting as I already spend enough time putting semicolons and other stuff (like trailing coma inside arrays or objects) just to have consistency so please comment when I ask you to and the test stuff is correctly placed and I vouched that the tests are working.